src/z3c/relationfield/relation.py

```diff
@@ -21,6 +21,10 @@ def __init__(self, to_id):
         self.__parent__ = None
         self.from_attribute = None
 
+    def __hash__(self):
+        """There should only be one RelationVaule per sort_key."""
+        return id(self)
```

Comment:
According to https://stackoverflow.com/questions/17192418/hash-function-in-python#17192858 the hash function is implemented as `id(obj) // 16`. Maybe we should use this here, too.